src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -171,6 +171,7 @@
     <PlatformManifestFileEntry Include="libmonosgen-2.0.dylib" IsNative="true" />
     <PlatformManifestFileEntry Include="llc" IsNative="true" />
     <PlatformManifestFileEntry Include="mono-aot-cross" IsNative="true" />
+    <PlatformManifestFileEntry Include="mono-aot-cross.exe" IsNative="true" FallbackFileVersion="6.0.0" />
```

Comment:
why do we need the `FallbackFileVersion`?

Reply:
Because the installer requires it for `.exe` files. Can't find the place in the code right now.

Reply:
It's not set for `createdump.exe` though :)

I think this actually verifies that the .exe has a non-0.0.0.0 file version, which I think should've been fixed by your `NativeVersion.rc` change. Can you double check this still happens?

Reply:
I think it was failing here IIRC: https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.SharedFramework.Sdk/src/GeneratePlatformManifestEntriesFromTemplate.cs#L84-L99